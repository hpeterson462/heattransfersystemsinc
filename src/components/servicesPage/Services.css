.serv-main {
  display: flex;
  flex-wrap: wrap;
  flex-direction: column;
  margin: auto;
}

.serviceCycle {
  margin: auto;
  text-align: center;
  width: 50%;
}

.serv-card-container {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-evenly;
  text-align: center;
  margin: auto;
  transform: translateY(20vh);
  visibility: hidden;
  transition: opacity 0.6s ease-out, transform 1.2s ease-out;
  background-image: linear-gradient(#E5E5E5, white);
  z-index: -10;
}

.serv-card-container.is-visible {
  opacity: 1;
  transform: none;
  visibility: visible;
}

.card-wrapper {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-evenly;
  margin: 30px;
  max-width: 80%;
}

.serv-card {
  margin: 20px;
  padding: 10px;
  width: 320px;
  line-height: 2rem;
  z-index: 10;
  border-radius: 2px;
  box-shadow: 0 0 5px 2px rgba(0, 0, 0, 0.3);
  background-color: #003366;
}

.serv-name {
  margin-bottom: 10px;
  font-size: 1.1rem;
  letter-spacing: 0.2rem;
  color: #F9FAFB;
}

.serv-img {
  width: 300px;
  height: 300px;
  border-radius: 2px;
}

.serv-p {
  text-align: justify;
  padding: 5px;
  hyphens: auto;
  font-family: 'Noto Sans', sans-serif;
  color: #F9FAFB;
}

@media only screen
  and (min-device-width : 20em) 
  and (max-device-width : 50em)
  and (-webkit-min-device-pixel-ratio: 3)
  and (orientation: portrait) {
    .circleArrow {
      width: 80%;
    }
}
