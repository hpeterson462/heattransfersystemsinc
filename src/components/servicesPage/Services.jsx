import React, { useEffect } from 'react';
import { useScrollAnimation } from '../../hooks/scrollAnimation';
import serviceCycle from '../../assets/serviceCycle.png';
import startUp from '../../assets/startUp.jpg';
import installation from '../../assets/installation.jpg';
import jobSite from '../../assets/jobSite.jpg';
import prevMaintenance from '../../assets/prevMaintenance.jpg';
import "aos/dist/aos.css";
import AOS from "aos";
import './Services.css';

export default function Services() {

  useEffect(() => {
    AOS.init();
    AOS.refresh();
  }, []);

  return (
    <>
      <section className="serv-main" id="services">
        <h1>
          Services
        </h1>

        <img
          className="serviceCycle"
          src={serviceCycle}
          alt="industrial heating cycle"
        />

        <h2>
          Keeping equipment running well is just as important as installing it correctly. Our service team is ready to assist you with the ongoing maintenance of your existing equipment. Whether a piece of equipment is brand new, or just new to you, we will get it running in no time.
        </h2>
      </section>

      <div className="serv-card-container" data-aos="fade-zoom-in" data-aos-offset="200" data-aos-easing="ease-in-sine" data-aos-duration="900">
        <div className="card-wrapper" data-aos="fade-zoom-in" data-aos-offset="200" data-aos-easing="ease-in-sine" data-aos-duration="900">
          <div className="serv-card">
            <div className="serv-name">
              Start Up
            </div>
            <img
              className="serv-img"
              src={startUp}
              alt="gas train"
            />
            <p className="serv-p">
              Our technicians are trained to perform the <b>initial start up</b> of your new combustion equipment <b>regardless of manufacturer</b>. You can be assured that the job will be done right and done efficiently.
          </p>
          </div>

          <div className="serv-card">
            <div className="serv-name">
              Installation
            </div>
            <img
              className="serv-img"
              src={installation}
              alt="installation of combustion equipment" />
            <p className="serv-p">
              Whether you have a <b>new piece of equipment</b> coming to your facility, or are <b>updating an existing combustion system</b>,  you can count on our team to complete the installation to the <b>highest standards</b>.
          </p>
          </div>

          <div className="serv-card">
            <div className="serv-name-2">
              Preventative Maintenance <br /> & Service
            </div>
            <img
              className="serv-img"
              src={prevMaintenance}
              alt="gas train"
            />
            <p className="serv-p">
              Ongoing, preventative maintenance can <b>reduce unplanned down time</b>, <b>minimize fuel costs</b>, <b>increase product quality</b>, and ensure your combustion <b>equipment is running safely</b>. However, if the unexpected does happen, our local team of service experts are ready to get you back into production quickly.
          </p>
            {/* <p className="serv-p">
              We offer a wide range of services:
            <br />
              <b>burner tuning</b>,
            <br /> regulatory <b>safety audits</b>,
            <br /> <b>troubleshooting & repairs</b>, and
            <br />proactive & preventative <b>maintenance</b>.
          </p> */}
          </div>

          {/* <div className="serv-card">
            <div className="serv-name">
              Service
            </div>
            <img
              className="serv-img"
              src={jobSite}
              alt="industrial heating system"
            />
            <p className="serv-p">
              We offer a wide range of services that include:
            <br />
              <b>burner tuning</b>,
            <br /> regulatory <b>safety audits</b>,
            <br /> <b>troubleshooting & repairs</b>, and
            <br />proactive & preventative <b>maintenance</b>.
          </p>
          </div> */}
        </div>
      </div>
    </>
  )
}
