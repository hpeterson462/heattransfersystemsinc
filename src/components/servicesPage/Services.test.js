import React from 'react';
import { cleanup, render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Services from './Services';

describe('Services component', () => {
  afterEach(() => cleanup());

  it('renders Services component', () => {
    const { asFragment } = render(<MemoryRouter><Services /></MemoryRouter>);

    expect(asFragment()).toMatchSnapshot();
  });
});
