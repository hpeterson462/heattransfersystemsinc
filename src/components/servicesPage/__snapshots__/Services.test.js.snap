// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services component renders Services component 1`] = `
<DocumentFragment>
  <section
    class="serv-main"
    id="services"
  >
    <h1>
      Services
    </h1>
    <img
      alt="industrial heating cycle"
      class="serviceCycle"
      src="serviceCycle.png"
    />
    <h2>
      Keeping equipment running well is just as important as installing it correctly. Our service team is ready to assist you with the ongoing maintenance of your existing equipment. Whether a piece of equipment is brand new, or just new to you, we will get it running in no time.
    </h2>
  </section>
  <div
    class="serv-card-container is-visible"
  >
    <div
      class="card-wrapper"
    >
      <div
        class="serv-card"
      >
        <div
          class="serv-name"
        >
          Start Up
        </div>
        <img
          alt="gas train"
          class="serv-img"
          src="startUp.jpg"
        />
        <p
          class="serv-p"
        >
          Our technicians are trained to perform the 
          <b>
            initial start up
          </b>
           of your new combustion equipment 
          <b>
            regardless of manufacturer
          </b>
          . You can be assured that the job will be done right and done efficiently.
        </p>
      </div>
      <div
        class="serv-card"
      >
        <div
          class="serv-name"
        >
          Installation
        </div>
        <img
          alt="installation of combustion equipment"
          class="serv-img"
          src="installation.jpg"
        />
        <p
          class="serv-p"
        >
          Whether you have a 
          <b>
            new piece of equipment
          </b>
           coming to your facility, or are 
          <b>
            updating an existing combustion system
          </b>
          ,  you can count on our team to complete the installation to the 
          <b>
            highest standards
          </b>
          .
        </p>
      </div>
      <div
        class="serv-card"
      >
        <div
          class="serv-name"
        >
          Preventative Maintenance
        </div>
        <img
          alt="gas train"
          class="serv-img"
          src="prevMaintenance.jpg"
        />
        <p
          class="serv-p"
        >
          Ongoing, preventative maintenance can 
          <b>
            reduce unplanned down time
          </b>
          , 
          <b>
            minimize fuel costs
          </b>
          , 
          <b>
            increase product quality
          </b>
          , and ensure your combustion 
          <b>
            equipment is running safely
          </b>
          . However, if the unexpected does happen, our local team of service experts are ready to get you back into production quickly.
        </p>
      </div>
      <div
        class="serv-card"
      >
        <div
          class="serv-name"
        >
          Service
        </div>
        <img
          alt="industrial heating system"
          class="serv-img"
          src="jobSite.jpg"
        />
        <p
          class="serv-p"
        >
          We offer a wide range of services that include:
          <br />
          <b>
            burner tuning
          </b>
          ,
          <br />
           regulatory 
          <b>
            safety audits
          </b>
          ,
          <br />
           
          <b>
            troubleshooting & repairs
          </b>
          , and
          <br />
          proactive & preventative 
          <b>
            maintenance
          </b>
          .
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
